src/jquery-sticky-header-footer.js

```diff
@@ -63,8 +63,7 @@
                 /**
                     Remove added DOM elements and plugin data
                  */
-                $.each([this.footerElement, this.headerElement], function(idx, val) {
-                    var element = that[val];
```

Comment:
Since we're iterating elements instead of strings (as I had it before), we don't have to look them up on the instance.